### ISSUE TITLE
Output is not obfuscated

### ISSUE THREAD

**Issue by soraphis** [mkaring/ConfuserEx#86]
confuserEx version: v1.2.0

I've a solution with multiple projects, that all build to dll's which i want to obfuscate.

I've added the following to my csproj file:
```
<Target Name="AfterBuild" Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
  <Exec Command="$(SolutionDir)..\ConfuserEx\Confuser.CLI.exe $(ProjectDir)confuser.crproj" />
</Target>
```

in my projects folder there is a `confuser.crproj` containing:
```
<?xml version="1.0" encoding="utf-8"?>
<project baseDir="obj\Release" outputDir="..\Release" xmlns="http://confuser.codeplex.com">
    <rule preset="none" pattern="true">
        <protection id="anti debug" />
        <protection id="anti dump" />
        <protection id="anti ildasm" />
        <protection id="anti tamper" />
        <protection id="constants" />
        <protection id="ctrl flow" />
        <protection id="invalid metadata" />
        <protection id="ref proxy" />
        <protection id="resources" />
    </rule>
    <module path="BitcoinCommon.dll" />
    <probePath>..\..\bin\Release</probePath>
</project>
```

building this results in an 
``` 
[ERROR] An IO error occurred, check if all input/output locations are readable/writable.
Exception: System.IO.IOException: Could not open file C:\...\Release\Common.dll for reading. Error: 00000003
```

so I've changed the confuser.crproj to read: `outputDir="..\Confused" baseDir="obj\Release"`

building this looks to be successful, but, looking with dotPeek in the Common.cs it lists: 

`[module: ConfusedBy("Confuser.Core 1.2.0+4110faee9d")]`

but looking in the code its not obfuscated at all

**Comment by mkaring** [comment]
That is a very common trap to stumble into. dotPeek locates the `*.pdb` file next to your assembly. This file contains the paths to the actual source files. All dotPeek does is show you your source files instead of what is actually decompiles.

You can spot this happening if you find your comments on the "decompiled" sources. Comments aren't part of the build output, so they can be decompiled.

Try deleting or renaming the `*.pdb` file. Once that is done you'll see the real decompiled sources.

**Comment by KvanTTT** [comment]
Duplicate of? #59

**Comment by mkaring** [comment]
> Duplicate of #59?

No. The obfuscation worked for him, because the watermark attribute was properly added.

**Comment by soraphis** [comment]
> Try deleting or renaming the *.pdb file. Once that is done you'll see the real decompiled sources.

I tried that before, but now again.
I deleted the *.pdb file  AND went into the dotPeek settings and turned off "Use sources from symbol files when available"

the output looked still the same. 
i cleared the dotpeek cache and now i get better results, being:

method bodys are replaced by `// ISSUE: unable to decompile the method`

I'm switched to ILSpy now, because clearing the cache all the time is tedious.
